### PR TITLE
remove _build directory

### PR DESCRIPTION
**Pull request by petrelharp**
I don't think we need the book in two places (`docs/` and `book/_build/html`).

### CODE REVIEW

**Comment by andrewkern** [comment]
yeah I think that's right. you might add the `binder/Dockerfile` to `.gitignore` on this PR too and kill this conflict that way

**Comment by petrelharp** [comment]
I don't think we can gitignore the Dockerfile, since that's what this "Update image tag" bot edits? See for instance 3f12c25d47d3032c2264e7939479431851f585e7 ?

File: .github/workflows/binder.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: pg_book
-on: [push] # You may want to trigger this Action on other things than a push.
+on:
+  push:
+    branches:
+      - main
```
Comment (petrelharp) [inline .github/workflows/binder.yml:5]: this should keep that from happening to PRs

**Comment by andrewkern** [comment]
so that's what the bot does, but only on the GH side of things, so I think it would be okay to ignore?

it's just a bit annoying that a local repo will *always* be out of sync with the upstream/main copy currently because of that GH action creating the docker file

**Comment by petrelharp** [comment]
I don't think that's how .gitignore works - either git tracks it or it doesn't; "on the github side" is on equal terms with the repo on your computer; so if it's in gitignore but still tracked then the only effect will be that the bot needs a `-f` to `git add` it.

**Pull request merged**